### PR TITLE
Update the baseline model to enable training

### PR DESCRIPTION
**Pull request by yy6linda**

### CODE REVIEW

File: examples/covid19-question-1/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ FROM python:3.5
 # Do not change
 LABEL challenge="covid19"
 
+#Enable training
```
Comment (tschaffter) [inline examples/covid19-question-1/Dockerfile]: Add space after \#

File: examples/covid19-question-1/README.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ Here we describe how to build and run locally an example model provided for Chal
...
+This example model takes 13 features that include age, clinical symptoms and vital signs. The feature selection refers to the research conducted by [Feng et al](https://www.medrxiv.org/content/10.1101/2020.03.19.20039099v1) and [Giuseppe et al](https://pubmed.ncbi.nlm.nih.gov/32348588/). First, we generate a feature set for each patient in the `/data` folder using the feature listed below and then we apply a 10-fold cross-validation logistic regression model on the feature set. Once the model is trained, we save the model file in the `/model`folder.
+
```
Comment (tschaffter) [inline examples/covid19-question-1/README.md:10]: Remove empty bullet point

File: examples/covid19-question-1/README.md [hunk 3]
```diff
@@ -47,38 +49,39 @@ This example model takes 13 features that include age, clinical symptoms and vit
...
 
-5. Run the dockerized model to generate predictions for the patients included in the synthetic dataset.
+6. Run the trained model on evaluation dataset and generate the prediction file.
 
     ```bash
     docker run \
-        -v $(pwd)/synthetic_data:/data:ro \
+        -v $(pwd)/synthetic_data/evaluation:/data:ro \
         -v $(pwd)/output:/output:rw \
         -v $(pwd)/scratch:/scratch:rw \
```
Comment (tschaffter) [inline examples/covid19-question-1/README.md:79]: Pending on decision to remove scratch for inference.

File: examples/covid19-question-1/train.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (tschaffter) [inline examples/covid19-question-1/train.sh]: Remove first empty line

**Review by tschaffter — CHANGES_REQUESTED** [review]

File: examples/covid19-question-1/README.md (hunk 2 above)
Comment (yy6linda) [reply examples/covid19-question-1/README.md:10]: Where is the empty bullet point?

File: examples/covid19-question-1/README.md (hunk 3 above)
Comment (yy6linda) [reply examples/covid19-question-1/README.md:79]: Should I keep it for now?

File: examples/covid19-question-1/README.md (hunk 2 above)
Comment (tschaffter) [reply examples/covid19-question-1/README.md:10]: My mistake

File: examples/covid19-question-1/README.md (hunk 3 above)
Comment (tschaffter) [reply examples/covid19-question-1/README.md:79]: Yes, an empty folder `scratch` will be available when running the script `infer.sh`.

**Pull request merged**